quadrat/inc/block-patterns.php

```diff
@@ -28,15 +27,18 @@ function quadrat_register_block_patterns() {
 				'listen-to-the-podcast',
 			);
 
+			if ( class_exists( 'WP_Block_Type_Registry' ) && \WP_Block_Type_Registry::get_instance()->is_registered( 'jetpack/subscriptions' ) ) {
```

Comment:
We should update the wpcom-only patterns in Blank Canvas to use this method, rather than loading them in their own separate `wpcom.php` file.